app/orders/page.tsx

```diff
@@ -166,7 +175,7 @@ export default function Orders() {
                           />
                         </TableCell>
                         <TableCell align="center">{row.bikeName}</TableCell>
-                        <TableCell align="center">{row.price}</TableCell>
+                        <TableCell align="center">{formattedPrice}</TableCell>
```

Comment:
```suggestion
                        <TableCell align="center">{row.price.toString()}</TableCell>
```
như này luôn là đc mà nhỉ, cần j thêm hàm

Reply:
Vừa thử thì thấy như này luôn cái format của column này không ăn